jetcd-resolver/src/test/java/com/coreos/jetcd/resolver/NameResolverTest.java

```diff
@@ -31,7 +29,7 @@ public void testDefaults() throws Exception {
     final URIResolverLoader loader = URIResolverLoader.defaultLoader();
     final SmartNameResolver resolver = new SmartNameResolver("etcd", Collections.emptyList(), loader);
 
-    assertThat(resolver.getResolvers().stream().anyMatch(DirectUriResolver.class::isInstance)).isTrue();
+    Assertions.assertThat(resolver.getResolvers().stream().anyMatch(DirectUriResolver.class::isInstance)).isTrue();
```

Comment:
any reason on changing imports to `import org.assertj.core.api.Assertions`?

Reply:
No done intentionally, guess the IDE did it while refactoring. Will fix it